### ISSUE TITLE
q3_gru.py add_prediction_op() 中dynamic_rnn应该取output作为值而不是state吧?

### ISSUE THREAD

**Issue by dulm** [hankcs/CS224n#11]
outputs, state = tf.nn.dynamic_rnn(cell, x, dtype=tf.float32)
output = outputs[:, -1]
preds = tf.sigmoid(output)

虽然gru中两者是等同的

**Comment by DingkunLiu** [comment]
Vanilla RNN中也是这样的，这里应该是不考虑LSTM的。
